modules/components/tag-input-form/tag-input-form.component.ts

```diff
@@ -10,27 +11,27 @@ export class TagInputForm implements OnInit, OnChanges {
     /**
      * @name onSubmit
      */
-    @Output() public onSubmit: EventEmitter<any> = new EventEmitter();
+    @Output() public onSubmit: EventEmitter<Event> = new EventEmitter();
 
     /**
      * @name onBlur
      */
-    @Output() public onBlur: EventEmitter<any> = new EventEmitter();
+    @Output() public onBlur: EventEmitter<FocusEvent> = new EventEmitter();
```

Comment:
I may have not used these in order to avoid errors with angular universal 🤔 @DanielKucal thoughts?

Reply:
I think things like those should be polyfilled in the server environment rather than in every project separately, e.g. like here: https://github.com/Angular-RU/angular-universal-starter/blob/master/server.ts#L11
We can eventually add quick polyfills like in tag-input.ts:
```
const DragEvent = (window as any).DragEvent;
```
@Gbuomprisco, what do you think?